SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -432,6 +432,21 @@ void FSpatialNetGUIDCache::RemoveEntityNetGUID(Worker_EntityId EntityId)
 		}
 	}
 
+	// Remove dynamically attached subobjects
+	USpatialActorChannel* Channel = SpatialNetDriver->GetActorChannelByEntityId(EntityId);
+	for (UObject* DynamicSubobject : Channel->CreateSubObjects)
+	{
+		if (FNetworkGUID* SubobjectNetGUID = NetGUIDLookup.Find(DynamicSubobject))
+		{
+			if (FUnrealObjectRef* SubobjectRef = NetGUIDToUnrealObjectRef.Find(*SubobjectNetGUID))
```

Comment:
Is there a better way to do these checks with less intendentation? I couldn't think of a way initially.

Reply:
We could ask for the NetGUID using the public functions, but then we'd have to check for validity. This seems okay.

Reply:
Sure